.github/workflows/ci.yml

```diff
@@ -1118,7 +1123,6 @@ ${{ hashFiles('.pre-commit-config.yaml') }}"
       - tests-mysql
       - tests-mssql
       - tests-postgres
-      - tests-kubernetes
```

Comment:
This is not needed - we do not have to wait for kurbernetes tests in order to decide if constraints should be pushed.